packages/goblint-cil/goblint-cil.1.7.3/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+authors: ["gabriel@kerneis.info"]
+maintainer: "ralf.vogler@gmail.com"
+homepage: "https://cil-project.github.io/cil/"
+bug-reports: "https://github.com/goblint/cil/issues/"
+dev-repo: "git+https://github.com/goblint/cil/"
+build: [
+  ["env" "FORCE_PERL_PREFIX=1" "./configure" "--prefix" prefix]
+  [make]
+]
+build-test: [
+  ["env" "VERBOSE=1" make "test"]
+]
+build-doc: [
+  [make "doc"]
+]
+install: [
+  make "install"
+]
+remove: [
+  ["env" "FORCE_PERL_PREFIX=1" "./configure" "--prefix" prefix]
+  [make "uninstall"]
+]
+depends: [
+  "ocamlfind"
+  "ocamlbuild" {build}
+  "hevea" {build & doc}
+  "hevea" {build & test}
```

Comment:
does this also need `conf-perl` given the `FORCE_PERL_PREFIX` in the build rules?